files/getOvfProperty.py

```diff
@@ -35,18 +38,20 @@ def main(argv):
     if len(argv) != 1:
         debug('usage: getOvfProperty.py <property_name')
         sys.exit(1)
-           
+
     ovf = get_ovf_properties()
-    
+
     try:
         res = ovf[argv[0]]
     except KeyError as err:
         debug(f'ovfProperty not found: {err}')
         sys.exit(1)
-    
+
+    # Strip enclosing quotes
+    res = re.sub(r'^(["\'])(.*)\1$', r'\2', res)
```

Comment:
question: this will only match enclosing (begin/end) quotes (" or ') but not if they'd appear within the string?

Reply:
That's correct - there are unit tests that cover the following scenarios:
- `"Hello World"` -> quotes are stripped
- `'Hello World'` -> quotes are striped
- `'Hello World"` -> quotes are not stripped
- `Hello World"` -> quotes are not stripped
- `& < > ' "` -> quotes are not stripped

However, having slept on it I realise there is an edge case where a password could legitimately have a leading and trailing quote. Passwords as a rule should pass through unaltered, so I'll update the PR with a suggestion as to how to handle this.